### PR TITLE
Don't run "kde-open" and "gnome-open" under Unix

### PR DESCRIPTION
**Pull request by konqoro**
All these utilities where replaced by xdg-open from package xdg-utils in Linux and the BSDs.
Checked: CentOS,  Solaris,  NetBSD,  OpenBSD,  FreeBSD' they all have it in their repos.

Testing done: Run it at a Linux system works.

### CODE REVIEW

**Comment by konqoro** [comment]
However I did not remove the last method to use the BROWSER env variable but I think it would be safe to do so.

**Comment by Yardanico** [comment]
@konqoro but sometimes xdg-open can be not installed

**Comment by konqoro** [comment]
Also since shellExecuteA was removed from winlean, couldn't this code be simplified even further?

**Comment by Araq** [comment]
Testing on a Linux machine is not good enough. Also: What about older Linux/disto versions?

**Comment by konqoro** [comment]
Gnome-open was faced-out in circa [2012](https://askubuntu.com/questions/101965/what-is-the-replacement-for-gnome-open-in-gnome) CentOS 6.9 has xdg-utils it in its repo. Kde-open must be a relic from KDE-3 times cause it doesn't exist in KDE4. To begin with, was this module even tested before it was commited or just had the names typed and maybe they will work?

File: lib/pure/browsers.nim [hunk 1]
```diff
@@ -21,24 +21,18 @@ proc openDefaultBrowser*(url: string) =
...
   when defined(windows):
-    when useWinUnicode:
-      var o = newWideCString("open")
-      var u = newWideCString(url)
-      discard shellExecuteW(0'i32, o, u, nil, nil, SW_SHOWNORMAL)
-    else:
-      discard shellExecuteA(0'i32, "open", url, nil, nil, SW_SHOWNORMAL)
+    var o = newWideCString("open")
+    var u = newWideCString(url)
+    discard shellExecuteW(0'i32, o, u, nil, nil, SW_SHOWNORMAL)
```
Comment (barcharcraz) [inline lib/pure/browsers.nim]: this seems unrelated to xdg-open? Is this because the uri should always be utf-8?

Comment (konqoro) [reply lib/pure/browsers.nim]: Yes unrelated. I removed the condition because shellExecuteA is removed thus dead code.
